packages/paste-core/components/in-page-navigation/src/InPageNavigationItem.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import type {BoxStyleProps, BoxProps} from '@twilio-paste/box';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+import {InPageNavigationContext} from './InPageNavigationContext';
+
+const getItemStyles = (currentPage: boolean, variant?: string): BoxStyleProps => {
+  return {
+    borderBottomColor: currentPage ? 'colorBorderPrimary' : 'transparent',
+    borderBottomStyle: 'solid',
+    borderBottomWidth: 'borderWidth20',
+    color: currentPage ? 'colorTextLink' : 'colorTextWeak',
+    flexBasis: variant === 'fullWidth' ? '50%' : undefined,
+    flexGrow: variant === 'fullWidth' ? 1 : undefined,
+    flexShrink: variant === 'fullWidth' ? 1 : undefined,
+    minWidth: 'sizeSquare130',
+    paddingBottom: 'space40',
+    paddingLeft: 'space20',
+    paddingRight: 'space20',
+    paddingTop: 'space40',
+    textAlign: 'center',
+    fontSize: 'fontSize30',
+    fontWeight: 'fontWeightSemibold',
+    overflow: 'hidden',
+    position: 'relative',
+    textOverflow: 'ellipsis',
+    transition: 'border-color 100ms ease, color 100ms ease',
+    whiteSpace: 'nowrap',
+    display: 'block',
+    textDecoration: 'none',
+    _hover: {
+      borderBottomColor: 'colorBorderPrimaryStronger',
+      color: 'colorTextLinkStronger',
+    },
+    _focus: {
+      borderBottomColor: 'colorBorderPrimaryStronger',
+      boxShadow: 'shadowFocus',
+      color: 'colorTextLinkStronger',
+      outline: 'none',
+    },
+  };
+};
+
+export interface InPageNavigationItemProps extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  currentPage?: boolean;
+  children: NonNullable<React.ReactNode>;
+  href: string;
+  ref?: any;
+  rel?: string; // do we need this? I don't understand how we're using it in Anchor
```

Comment:
I added this prop bc it's in our Anchor component, but when I looked at Anchor, I didn't truly understand what we did with it there. The main question is, do we need this prop in InPageNavigationItem?

Reply:
We probably should, its a HTML attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#:~:text=The%20rel%20attribute%20defines%20the,which%20the%20attribute%20is%20found.

Usually I see people set it to noreferrer.. 

However, I think it should be included in `React.AnchorHTMLAttributes`- not sure why it would need to be specified here too.

Reply:
update: 
- `rel` and `target` are included in `React.AnchorHTMLAttributes` as @shleewhite pointed out above, so it has been removed from the props
- `href`, though also included in `React.AnchorHTMLAttributes`, is optional there while we want it to be required here, so it is declared here to be required.
- `ref` is not necessary to add to our props, so it has been removed. @TheSisb and I tested this out by removing this line and adding a ref to a story. It worked.